Apps/phcofensetriage/cofensetriage.json

```diff
@@ -1385,8 +1392,7 @@
                     "order": 8
                 }
             },
-            "output": [
-                {
+            "output": [{
```

Comment:
This format is inconsistent with all of our apps

Reply:
Fixed the inconsistency